test/rebar3_hex_SUITE.erl

```diff
@@ -14,3 +27,31 @@ repo_opt(_Config) ->
 init_test(_Config) ->
     {ok, State} = rebar3_hex:init(rebar_state:new()),
     ?assertEqual(state_t, element(1, State)).
+
+%% Smoke test to ensure we don't crash
+help_test(_Config) -> 
+    %% Silent output during our tests
```

Comment:
This is really a brute force method for silencing output... I tried setting the group leader to a dummy proc but had issues... maybe there's a good pattern for this that I'm not aware of?